example/manual.ts

```diff
@@ -7,16 +7,22 @@ import {
   route
 } from '../packages/ton'
 import * as logger from '../packages/logger'
+import { createStaticHandler } from '../packages/static/src'
 
 const host = '0.0.0.0'
 const port = 3000
 
 async function main() {
   const app = createApp()
+  const staticServer = createStaticHandler({
+    root: `${process.cwd()}/example`
+  })
   route(app, 'get', '/', (req, res) => send(res, 200, 'TonJS'))
   route(app, 'get', '/redirect', (req, res) =>
     redirect(res, 302, 'https://tonjs.com')
   )
+  route(app, 'get', '/public/*', staticServer)
+
```

Comment:
restore the old version, and create the new example about static named `example/static.ts` and serve the `example/public` folder with `@tonjs/bin`, plz.